grunt/jshint.js

```diff
@@ -4,7 +4,8 @@ module.exports = function(grunt) {
     build: "tmp/wrapped.js",
     options: {
       // this is for data.js which sometimes has commas on the following line
-      laxcomma: true
+      laxcomma: true,
+      reporterOutput: ""
```

Comment:
I faced this bug: https://github.com/gruntjs/grunt-contrib-jshint/issues/256 when running `jshint` task ![terminal](http://dl.dropboxusercontent.com/u/16789281/Screenshot/img-2016-09-24-00-44-13.png)
So I needed to update _grunt-contrib-jshint_ package or add _this_ option to finish the build.